src/read/mod.rs

```diff
@@ -254,7 +254,9 @@ where
         Some(self.get_raw(key.borrow())?.map_ref(Values::user_friendly))
     }
 
-    /// Returns a guarded reference to the first value corresponding to the key.
+    /// Returns a guarded reference to _one_ value corresponding to the key.
+    /// This is mostly intended for use when you are working with no more than one value per key.
```

Comment:
Super nitpicky, but could you leave an empty line after the first sentence? Makes it a little easier to visually parse out the "important" parts of the docs.

Reply:
And same for the other `get_one`.